opencolorio_config_aces/config/studio/generate/resources/OpenColorIO-Config-ACES CG and Studio Transforms - v1.0.0-rc1 - Studio Config - Mapping.csv

```diff
@@ -4,17 +4,18 @@ Ordering,Legacy Name,ACEStransformID,CLFtransformID,Interface,BuiltinTransform S
 100,ACES - ACEScg,urn:ampas:aces:transformId:v1.5:ACEScsc.Academy.ACEScg_to_ACES.a1.0.3,,ColorSpace,ACEScg_to_ACES2065-1,"acescg,lin_ap1",scene-linear,ACES
 200,Input - ADX - ADX10,urn:ampas:aces:transformId:v1.5:ACEScsc.Academy.ADX10_to_ACES.a1.0.3,,ColorSpace,ADX10_to_ACES2065-1,adx10,log,ACES
 200,Input - ADX - ADX16,urn:ampas:aces:transformId:v1.5:ACEScsc.Academy.ADX16_to_ACES.a1.0.3,,ColorSpace,ADX16_to_ACES2065-1,adx16,log,ACES
-203,Input - ARRI - Curve - V3 LogC (EI800),,,NamedTransform,,crv_logc3ei800,log,ARRI
-203,Input - ARRI - Linear - ALEXA Wide Gamut,,,ColorSpace,,lin_awg,scene-linear,ARRI
-203,Input - ARRI - V3 LogC (EI800) - Wide Gamut,urn:ampas:aces:transformId:v1.5:IDT.ARRI.Alexa-v3-logC-EI800.a1.v2,,ColorSpace,,logc3ei800_awg,log,ARRI
-204,Input - ARRI - Curve - LogC4,,,NamedTransform,,crv_logc4,log,
-204,Input - ARRI - Linear - ARRI Wide Gamut 4,,,ColorSpace,,lin_awg4,scene-linear,
-204,Input - ARRI LogC4,,,ColorSpace,,logc4_awg4,log,
+203,Input - ARRI - Curve - V3 LogC (EI800),,urn:aswf:ocio:transformId:1.0:ARRI:Input:ARRI_LogC3_Curve_EI800_to_Linear:1.0,NamedTransform,,crv_logc3ei800,log,ARRI
+203,Input - ARRI - Linear - ALEXA Wide Gamut,,urn:aswf:ocio:transformId:1.0:ARRI:Input:Linear_ARRI_Wide_Gamut_3_to_ACES2065-1:1.0,ColorSpace,,lin_awg,scene-linear,ARRI
+203,Input - ARRI - V3 LogC (EI800) - Wide Gamut,urn:ampas:aces:transformId:v1.5:IDT.ARRI.Alexa-v3-logC-EI800.a1.v2,urn:aswf:ocio:transformId:1.0:ARRI:Input:ARRI_LogC3_EI800_to_ACES2065-1:1.0,ColorSpace,,logc3ei800_awg,log,ARRI

```

Comment:
There are some minor variations in some of the names.  The CLFs have a "name" appearing multiple times in different fields.  My understanding is that the generator is generating the config color space name by taking the CLF transform "id" attribute and doing some substitutions (e.g. replace "_" with a space).

However, this means that the config color space name is sometimes slightly different from the "name" attribute or the "InputDescriptor" element.  These latter "names" are more obviously human readable strings and I'm worried that some of the camera partners may have assumed that those would be what becomes the color space name.  We should probably review these at our next meeting.

For example: 
config name: "ARRI LogC3 EI800",  CLF InputDescriptor = "ARRI LogC3 (EI800)"
config name: "CanonLog3 CinemaGamut D55",  CLF InputDescriptor = "Canon Log 3 Cinema Gamut (Daylight)"

Reply:
If you are reading the CLF file anyway, perhaps it would be better to take the InputDescriptor for the color space name and avoid needing to convert the ID into the name with various substitutions?  (I could post a bit of code to read that using OCIO, let me know if it would be useful.)

Reply:
> We should probably review these at our next meeting.

Yes!